<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-3QHODDFO" ignored="0" name="Calculator" tests="49" failures="6" timestamp="2020-07-13T18:30:27 ALMT" time="0.155" errors="0">
  <testcase name="testDiv" time="0.005" classname="com.epam.tat.module4.ArithmeticFunctionality.DivDoubleTest"/>
  <testcase name="sumLongTest" time="0.004" classname="com.epam.tat.module4.ArithmeticFunctionality.SumLongTest"/>
  <testcase name="testDiv" time="0.005" classname="com.epam.tat.module4.ArithmeticFunctionality.DivLongTest"/>
  <testcase name="sumLongTest" time="0.004" classname="com.epam.tat.module4.ArithmeticFunctionality.SumDoubleTest"/>
  <testcase name="testDiv" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.DivLongTest"/>
  <testcase name="sumLongTest" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.SumDoubleTest"/>
  <testcase name="testDiv" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.DivDoubleTest"/>
  <testcase name="testDiv" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.DivLongTest"/>
  <testcase name="sumLongTest" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.SumDoubleTest"/>
  <testcase name="testDiv" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.DivDoubleTest"/>
  <testcase name="testMult" time="0.001" classname="com.epam.tat.module4.ArithmeticFunctionality.MultLongTest"/>
  <testcase name="testPow" time="0.001" classname="com.epam.tat.module4.ArithmeticFunctionality.PowTest"/>
  <testcase name="testMult" time="0.001" classname="com.epam.tat.module4.ArithmeticFunctionality.MultDoubleTest"/>
  <testcase name="testPow" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.PowTest"/>
  <testcase name="testMult" time="0.001" classname="com.epam.tat.module4.ArithmeticFunctionality.MultDoubleTest"/>
  <testcase name="testMult" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.MultLongTest"/>
  <testcase name="testMult" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.MultDoubleTest"/>
  <testcase name="testMult" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.MultLongTest"/>
  <testcase name="testPow" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.PowTest"/>
  <testcase name="subLongTest" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.SubLongTest"/>
  <testcase name="testSqrt" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.SqrtTest"/>
  <testcase name="subLongTest" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.SubLongTest"/>
  <testcase name="testSqrt" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.SqrtTest"/>
  <testcase name="subLongTest" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.SubLongTest"/>
  <testcase name="isNegativeTest" time="0.0" classname="com.epam.tat.module4.LogicalFunctionality.IsNegativeTest"/>
  <testcase name="isNegativeTest" time="0.0" classname="com.epam.tat.module4.LogicalFunctionality.IsNegativeTest"/>
  <testcase name="isNegativeTest" time="0.0" classname="com.epam.tat.module4.LogicalFunctionality.IsNegativeTest"/>
  <testcase name="subLongTest" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.SubDoubleTest"/>
  <testcase name="isPositiveTest" time="0.0" classname="com.epam.tat.module4.LogicalFunctionality.IsPositiveTest"/>
  <testcase name="isPositiveTest" time="0.0" classname="com.epam.tat.module4.LogicalFunctionality.IsPositiveTest"/>
  <testcase name="isPositiveTest" time="0.0" classname="com.epam.tat.module4.LogicalFunctionality.IsPositiveTest"/>
  <testcase name="subLongTest" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.SubDoubleTest"/>
  <testcase name="subLongTest" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.SubDoubleTest"/>
  <testcase name="subLongTest" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.SubDoubleTest"/>
  <testcase name="TgTest" time="0.0" classname="com.epam.tat.module4.TrigonometricFunctionality.TgTest"/>
  <testcase name="SinTest" time="0.002" classname="com.epam.tat.module4.TrigonometricFunctionality.SinTest">
    <failure type="java.lang.AssertionError" message="incorrect sin result expected [0.8939966636] but found [0.8939966636005579]">
      <![CDATA[java.lang.AssertionError: incorrect sin result expected [0.8939966636] but found [0.8939966636005579]
at com.epam.tat.module4.TrigonometricFunctionality.SinTest.SinTest(SinTest.java:14)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- SinTest -->
  <testcase name="SinTest" time="0.0" classname="com.epam.tat.module4.TrigonometricFunctionality.SinTest"/>
  <testcase name="TgTest" time="0.0" classname="com.epam.tat.module4.TrigonometricFunctionality.TgTest">
    <failure type="java.lang.AssertionError" message="incorrect tg result expected [0.0] but found [NaN]">
      <![CDATA[java.lang.AssertionError: incorrect tg result expected [0.0] but found [NaN]
at com.epam.tat.module4.TrigonometricFunctionality.TgTest.TgTest(TgTest.java:14)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- TgTest -->
  <testcase name="SinTest" time="0.0" classname="com.epam.tat.module4.TrigonometricFunctionality.SinTest"/>
  <testcase name="testDivByZero" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.DivLongTest"/>
  <testcase name="testDivByZero" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.DivDoubleTest"/>
  <testcase name="CtgTest" time="0.001" classname="com.epam.tat.module4.TrigonometricFunctionality.CtgTest"/>
  <testcase name="CtgTest" time="0.0" classname="com.epam.tat.module4.TrigonometricFunctionality.CtgTest">
    <failure type="java.lang.AssertionError" message="incorrect ctg result expected [-0.50120278338] but found [1.0]">
      <![CDATA[java.lang.AssertionError: incorrect ctg result expected [-0.50120278338] but found [1.0]
at com.epam.tat.module4.TrigonometricFunctionality.CtgTest.CtgTest(CtgTest.java:14)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- CtgTest -->
  <testcase name="CtgTest" time="0.001" classname="com.epam.tat.module4.TrigonometricFunctionality.CtgTest"/>
  <testcase name="sumLongTest" time="0.0" classname="com.epam.tat.module4.ArithmeticFunctionality.SumLongTest"/>
  <testcase name="sumLongTest" time="0.001" classname="com.epam.tat.module4.ArithmeticFunctionality.SumLongTest"/>
  <testcase name="CosTest" time="0.008" classname="com.epam.tat.module4.TrigonometricFunctionality.CosTest">
    <failure type="java.lang.AssertionError" message="incorrect cos result expected [1.0] but found [0.9589157234143065]">
      <![CDATA[java.lang.AssertionError: incorrect cos result expected [1.0] but found [0.9589157234143065]
at com.epam.tat.module4.TrigonometricFunctionality.CosTest.CosTest(CosTest.java:14)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- CosTest -->
  <testcase name="CosTest" time="0.0" classname="com.epam.tat.module4.TrigonometricFunctionality.CosTest">
    <failure type="java.lang.AssertionError" message="incorrect cos result expected [0.5] but found [-0.3048106211022167]">
      <![CDATA[java.lang.AssertionError: incorrect cos result expected [0.5] but found [-0.3048106211022167]
at com.epam.tat.module4.TrigonometricFunctionality.CosTest.CosTest(CosTest.java:14)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- CosTest -->
  <testcase name="CosTest" time="0.0" classname="com.epam.tat.module4.TrigonometricFunctionality.CosTest">
    <failure type="java.lang.AssertionError" message="incorrect cos result expected [1.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: incorrect cos result expected [1.0] but found [0.0]
at com.epam.tat.module4.TrigonometricFunctionality.CosTest.CosTest(CosTest.java:14)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- CosTest -->
</testsuite> <!-- Calculator -->
